slices/systemd.yaml

```diff
@@ -0,0 +1,635 @@
+      /usr/bin/systemd-run:
+      /usr/bin/systemd-socket-activate:
+      /usr/bin/systemd-stdio-bridge:
+      /usr/bin/systemd-sysext:
+      /usr/bin/systemd-sysusers:
+      /usr/bin/systemd-tmpfiles:
+      /usr/bin/systemd-tty-ask-password-agent:
+      /usr/bin/systemd-umount:
+      /usr/bin/timedatectl:
+      /usr/bin/varlinkctl:
+      /usr/lib/systemd/systemd:
+      /usr/lib/systemd/systemd-backlight:
+      /usr/lib/systemd/systemd-battery-check:
+      /usr/lib/systemd/systemd-binfmt:
+      /usr/lib/systemd/systemd-boot-check-no-failures:
+      /usr/lib/systemd/systemd-bsod:
+        arch: [amd64, arm64, riscv64, ppc64el, armhf, s390x]
+      /usr/lib/systemd/systemd-cgroups-agent:
+      /usr/lib/systemd/systemd-cryptsetup:
+      /usr/lib/systemd/systemd-executor:
+      /usr/lib/systemd/systemd-fsck:
+      /usr/lib/systemd/systemd-fsckd:
+      /usr/lib/systemd/systemd-growfs:
+      /usr/lib/systemd/systemd-hibernate-resume:
+      /usr/lib/systemd/systemd-hostnamed:
+      /usr/lib/systemd/systemd-initctl:
+      /usr/lib/systemd/systemd-integritysetup:
+      /usr/lib/systemd/systemd-journald:
+      /usr/lib/systemd/systemd-localed:
+      /usr/lib/systemd/systemd-logind:
+      /usr/lib/systemd/systemd-makefs:
+      /usr/lib/systemd/systemd-measure:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-modules-load:
+      /usr/lib/systemd/systemd-network-generator:
+      /usr/lib/systemd/systemd-networkd:
+      /usr/lib/systemd/systemd-networkd-wait-online:
+      /usr/lib/systemd/systemd-pcrextend:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-pstore:
+      /usr/lib/systemd/systemd-quotacheck:
+      /usr/lib/systemd/systemd-random-seed:
+      /usr/lib/systemd/systemd-remount-fs:
+      /usr/lib/systemd/systemd-reply-password:
+      /usr/lib/systemd/systemd-rfkill:
+      /usr/lib/systemd/systemd-shutdown:
+      /usr/lib/systemd/systemd-sleep:
+      /usr/lib/systemd/systemd-socket-proxyd:
+      /usr/lib/systemd/systemd-storagetm:
+      /usr/lib/systemd/systemd-sulogin-shell:
+      /usr/lib/systemd/systemd-sysctl:
+      /usr/lib/systemd/systemd-sysroot-fstab-check:
+      /usr/lib/systemd/systemd-sysupdate:
+      /usr/lib/systemd/systemd-sysv-install:
+      /usr/lib/systemd/systemd-time-wait-sync:
+      /usr/lib/systemd/systemd-timedated:
+      /usr/lib/systemd/systemd-tpm2-setup:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-update-done:
+      /usr/lib/systemd/systemd-update-utmp:
+      /usr/lib/systemd/systemd-user-runtime-dir:
+      /usr/lib/systemd/systemd-user-sessions:
+      /usr/lib/systemd/systemd-veritysetup:
+      /usr/lib/systemd/systemd-volatile-root:
+      /usr/lib/systemd/systemd-xdg-autostart-condition:
+
+  config:
+    contents:
+      /etc/modules-load.d/modules.conf:
+      /etc/sysctl.d/99-sysctl.conf:
```

Comment:
Are these broken symlinks?

```
root@de86f5969e22:~/systemd# file ./etc/modules-load.d/modules.conf ./etc/sysctl.d/99-sysctl.conf 
./etc/modules-load.d/modules.conf: broken symbolic link to ../modules
./etc/sysctl.d/99-sysctl.conf:     broken symbolic link to ../sysctl.conf
```

Reply:
yes, they are moved to an `extras` slice, they are broken for a reason since the files are either generated and/or come from other systemd-packages.